src/coreclr/src/tools/crossgen2/ILCompiler.Reflection.ReadyToRun/x86/InfoHdr.cs

```diff
@@ -276,7 +276,7 @@ public static InfoHdrSmall DecodeHeader(byte[] image, ref int offset, int codeLe
                                 break;
 
                             case (byte)InfoHdrAdjust.NEXT_OPCODE:
-                                encoding = (byte)(image[offset++] & (int)InfoHdrAdjustConstants.ADJ_ENCODING_MAX);
+                                encoding = (byte)((nextByte = image[offset++]) & (int)InfoHdrAdjustConstants.ADJ_ENCODING_MAX);
```

Comment:
A nit - it would be easier to read this if you moved the nextByte assignment out of this expression to the previous line. Similar to what the related c++ code looks like.
```suggestion
                                nextByte = image[offset++];
                                encoding = (byte)(nextByte & (int)InfoHdrAdjustConstants.ADJ_ENCODING_MAX);
```